### ISSUE TITLE
powerline not working for tmux 

### ISSUE THREAD

**Issue by typekpb** [powerline/powerline#688]
I'm using konsole + bash.

I was able to setup powerline in bash (following: https://powerline.readthedocs.org/en/latest/overview.html#bash-prompt), however it doesn't work for me for tmux statusline (following: https://powerline.readthedocs.org/en/latest/overview.html#tmux-statusline).

I added: 
    source ~/.vim/bundle/powerline/powerline/bindings/tmux/powerline.conf

to my `tmux.conf`

however my tmux statusbar looks the old way.

when invoking the command in tmux:

```
source ~/.vim/bundle/powerline/powerline/bindings/tmux/powerline.conf
```

I get following output:

```
bash: if-shell: Befehl nicht gefunden...
-bash: set: -g: Ungültige Option
set: Gebrauch: set [-abefhkmnptuvxBCHP] [-o option-name] [--] [arg ...]
-bash: set: -g: Ungültige Option
set: Gebrauch: set [-abefhkmnptuvxBCHP] [-o option-name] [--] [arg ...]
-bash: set: -g: Ungültige Option
set: Gebrauch: set [-abefhkmnptuvxBCHP] [-o option-name] [--] [arg ...]
-bash: set: -g: Ungültige Option
set: Gebrauch: set [-abefhkmnptuvxBCHP] [-o option-name] [--] [arg ...]
-bash: set: -g: Ungültige Option
set: Gebrauch: set [-abefhkmnptuvxBCHP] [-o option-name] [--] [arg ...]
-bash: set: -g: Ungültige Option
set: Gebrauch: set [-abefhkmnptuvxBCHP] [-o option-name] [--] [arg ...]
-bash: set: -g: Ungültige Option
set: Gebrauch: set [-abefhkmnptuvxBCHP] [-o option-name] [--] [arg ...]
-bash: set: -g: Ungültige Option
set: Gebrauch: set [-abefhkmnptuvxBCHP] [-o option-name] [--] [arg ...]
-bash: set: -g: Ungültige Option
set: Gebrauch: set [-abefhkmnptuvxBCHP] [-o option-name] [--] [arg ...]
-bash: set: -g: Ungültige Option
set: Gebrauch: set [-abefhkmnptuvxBCHP] [-o option-name] [--] [arg ...]
-bash: set: -g: Ungültige Option
set: Gebrauch: set [-abefhkmnptuvxBCHP] [-o option-name] [--] [arg ...]
bash: set-window-option: Befehl nicht gefunden...
bash: set-window-option: Befehl nicht gefunden...
bash: set-window-option: Befehl nicht gefunden...
bash: set-window-option: Befehl nicht gefunden...
bash: set-window-option: Befehl nicht gefunden...
```

So basically it can't find `if-shell` command, moreover has problems with `set -g` argument.

How should I proceed?
